.github/workflows/store-ffmpeg-builds.yml

```
@@ -20,21 +20,27 @@ jobs:
         run: |
           set -eu
           sudo apt update -q
-          sudo apt install -y -q s3cmd
+          sudo apt install -y -q jq s3cmd
 
-      - name: Download FFmpeg
-        run: |
-          wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-${{ matrix.arch }}-static.tar.xz
-
-      - name: Unpack FFmpeg
-        run: |
-          tar xvf ffmpeg-release-${{ matrix.arch }}-static.tar.xz
+      - name: Get FFmpeg download URL
+        run: >
+          curl -fL
+          -H "Accept: application/vnd.github+json"
+          -H "X-GitHub-Api-Version: 2022-11-28"
+          https://api.github.com/repos/BtbN/FFmpeg-Builds/releases
+          | jq -r '
+            [
+              .[1].assets[]
+              | select(.name | contains("${{ matrix.arch }}-gpl-"))
```

They started creating `winarm64` builds. Let's filter for `linux` as well:
```suggestion
              | select(.name | contains("${{ matrix.arch }}-gpl-"))
              | select(.name | contains("linux"))
```